grove/amplification/grover.py

```diff
@@ -20,26 +20,31 @@
 import pyquil.quil as pq
 from pyquil.gates import H
 from pyquil.quilbase import Qubit
+from pyquil.api import JobConnection
```

Comment:
where is this import used?

Reply:
interesting, nowhere. Pycharm didn't tell me. :(